python/foglamp/services/core/routes.py

```diff
@@ -175,6 +176,12 @@ def setup(app):
     app.router.add_route('PUT', '/foglamp/notification/{notification_name}', notification.put_notification)
     app.router.add_route('DELETE', '/foglamp/notification/{notification_name}', notification.delete_notification)
 
+    # Snapshot config
+    app.router.add_route('GET', '/foglamp/config/snapshot', snapshot_config.get_snapshot)
```

Comment:
let's make the route /foglamp/snapshot/category - that should solve the issue with conflicts with the existing API and avoids the mixing of category and config. 

We can make all snaps calls use /foglamp/snapshot/...